ecs-logs-collector.sh

```diff
@@ -213,11 +213,11 @@ pack() {
   try "archive gathered log information"
 
   local tar_bin
-  tar_bin="`which tar 2>/dev/null`"
+  tar_bin="$(which tar 2>/dev/null)"
   [ -z "${tar_bin}" ] && warning "TAR archiver not found, please install a TAR archiver to create the collection archive. You can still view the logs in the collect folder."
 
-  cd ${curdir}
-  ${tar_bin} -czf ${infodir}.tgz ${infodir} > /dev/null 2>&1
+  cd "$curdir" || echo "cd failed."; exit 
```

Comment:
This will need some braces around it to prevent the unconditional exit:
```suggestion
  cd "$curdir" || { echo "cd failed."; exit 1; }
```